.github/workflows/test.yaml

```diff
@@ -74,20 +74,47 @@ jobs:
         run: |
           # Start a sandbox conductor and run it in the background
           nix develop .#ci -c bash -c "hc s clean && echo "1234" | hc s --piped create && echo "1234" | hc s --piped -f 8888 run &"
-          
+
           # Run the scenario for 5 seconds
           RUST_LOG=info nix run .#zome_call_single_value -- --connection-string ws://localhost:8888 --duration 5 --no-progress
-          
+
+          pkill hc && pkill holochain && pkill lair-keystore
+
+      - name: Smoke test - zome_call_single_value with CHC enabled
```

Comment:
This scenario doesn't write data, is it intentional to have CHC enabled on it? I don't think it will touch CHC at all, so I guess that's a kind of baseline but I'd argue that we already have that from running one of the scenarios that writes data and doesn't have the CHC enabled

Reply:
Hmmm from my testing it did write to the CHC, but not as much removed